src/pop/c/entities/btcblock.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2019-2021 Xenios SEZC
+// https://www.veriblock.org
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php.
+
+#include <memory>
+
+#include "veriblock/pop/c/entities/btcblock.h"
+#include "veriblock/pop/entities/btcblock.hpp"
+
+POP_ENTITY_STRUCT_NAME(btc_block) {
+  std::shared_ptr<altintegration::BtcBlock> ref;
+};
+
+POP_ENTITY_NEW_SIGNATURE(btc_block) { return new POP_ENTITY_NAME(btc_block)(); }
+
+POP_ENTITY_FREE_SIGNATURE(btc_block) {
```

Comment:
hm. I think it is better to explicitly write func name... or at least expose `self` arg

Reply:
fixed